Makefile

```diff
@@ -4,6 +4,8 @@ VERSION := 18.08.8
 URL_BASE :=  https://download.schedmd.com/$(NAME)
 SOURCE   = $(URL_BASE)/$(NAME)-$(VERSION).tar.$(SRC_EXT)
 
+MOCK_OPTIONS := --with=mysql
```

Comment:
This needs to be interpolated into `COMMON_RPM_ARGS` also so that `make rpms` uses the same flag.

Somewhere in one of the RPM build projects is a `Makefile_packaging.mk` that takes a single variable and interpolates it into both of the `COMMON_RPM_ARGS` and `MOCK_OPTIONS`.  But I cannot seem to find it now.

In any case, it's probably useful if the `Makefile_packaging.mk` would interpolate a single variable called, perhaps,
`RPM_BUILD_OPTIONS` into both `MOCK_OPTIONS` and `COMMON_RPM_ARGS`.

Reply:
RPM_BUILD_OPTIONS is referenced currently one place in the Makefile_packaging.mk.

I will add it to the mockbuild and srpm recipes in the same way to be consistent, as it is possible that there may be mock options that are not rpmbuild options.